src/components/Rockets/Rockets.js

```diff
@@ -19,7 +19,7 @@ const Rockets = () => {
               <h3>{name}</h3>
               <p className="reserve-sect">
                 {reserved === true && (
-                <span className="reverved">
+                <span className="reserved">
                   Reserved
                 </span>
```

Comment:
[REQUIRED] Good job implementing the switch badges, it works perfectly. However, I suggest that you style the buttons for add and cancel reservations according to the project requirements.